.travis.yml

```diff
@@ -22,6 +22,7 @@ after_success:
 - DEPLOY_BUILD_READY=true
 after_script:
 - make clean
+- ./tools/travis/docker/publish.sh apache wskdeploy ${TRAVIS_TAG}
```

Comment:
We need to move this line into a new section of provider.

deploy:
-provider: releases
 ...
 ...
- provider: script
  script: "./tools/travis/docker/publish.sh apache wskdeploy ${TRAVIS_TAG}"
  on:
    tags: true
    repo: apache/incubator-openwhisk-wskdeploy
    condition: "$DEPLOY_BUILD_READY = true"

Reply:
thanks -- this wasn't originally working, until I realized i needed skip_cleanup = true.  Should be good to go now.